deployer/handlers.go

```diff
@@ -58,7 +59,12 @@ func Validate(awsc aws.Clients) DeployHandler {
 func Lock(awsc aws.Clients) interface{} {
 	return func(ctx context.Context, release *Release) (*Release, error) {
 		// returns LockExistsError, LockError
-		return release, release.GrabLocks(awsc.S3(release.AwsRegion, nil, nil))
+		err := release.GrabLocks(
```

Comment:
Hmm, I would have expected you to prefer the single-statement `return ...` form.

Reply:
It's nitpicky, I admit, but I normally prefer `return foo(...), err` when using `foo`'s result as the function's "main" return value, but this form when using `foo`'s result as the function's error value.